apiendpoint/api_endpoint.go

```diff
@@ -0,0 +1,234 @@
+// Package apiendpoint provides a lightweight API framework for use with River
+// UI. It lets API endpoints be defined, then mounted into an http.ServeMux.
```

Comment:
Can probably drop the river UI reference and generalize this description.

Reply:
Done.